app/models/user.rb

```diff
@@ -456,6 +456,16 @@ def close_account!
     AccountDeletion.create(:person => self.person)
   end
 
+  def lock_account!
+    self.locked_at = Time.now
+    self.save
+  end
```

Comment:
@jhass 
The code is:
>>unless user.access_locked?
>>  #Lock_account code
>>else
>> #Unlock_account code

So when user account is closed then: 
user.access_locked? will return true, hence else part will get executed and "Unlock Account" will appear. I will have to write the code for user.access_unlocked? in "user.rb". 
Please correct me if I am wrong.